Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM node:lts-alpine
+
+RUN \
+  apk update && \
+  apk add --no-cache \
+    python3 \
+    && \
+  cp /usr/bin/python3 /usr/bin/python && \
+  pip3 install \
+    pyspf \
+    dnspython \
+    dkimpy
+
+WORKDIR /app
+
+COPY package*.json ./
+COPY yarn.lock yarn.lock
+
+RUN npm i -g pm2 && npx yarn --prod
+
+COPY index.js index.js
```

Comment:
I'd go for the inverse strategy, ignore files that aren't needed in the image into the `.dockerignore` and then do a ` COPY . ./`